hummingbot/connector/derivative/coinflex_perpetual/coinflex_perpetual_derivative.py

```diff
@@ -0,0 +1,1391 @@
+        )
+        return order_id
+
+    async def cancel_all(self, timeout_seconds: float):
+        """
+        The function that is primarily triggered by the ExitCommand that cancels all InFlightOrder's
+        being tracked by the Client Order Tracker. It confirms the successful cancellation
+        of the orders.
+
+        Parameters
+        ----------
+        timeout_seconds:
+            How long to wait before checking whether the orders were cancelled
+        """
+        incomplete_orders = [order for order in self._client_order_tracker.active_orders.values() if not order.is_done]
+        tasks = [self._execute_cancel(order.trading_pair, order.client_order_id) for order in incomplete_orders]
+        order_id_set = set([o.client_order_id for o in incomplete_orders])
+        successful_cancellations = []
+
+        try:
+            async with timeout(timeout_seconds):
+                cancellation_results = await safe_gather(*tasks, return_exceptions=True)
+                for cr in cancellation_results:
+                    if isinstance(cr, Exception):
+                        continue
+                    if isinstance(cr, dict) and "clientOrderId" in cr:
+                        client_order_id = cr.get("clientOrderId")
+                        order_id_set.remove(client_order_id)
+                        successful_cancellations.append(CancellationResult(client_order_id, True))
+        except Exception:
+            self.logger().network(
+                "Unexpected error canceling orders.",
+                exc_info=True,
+                app_warning_msg="Failed to cancel order with Coinflex Perpetual. Check API key and network connection."
+            )
+        failed_cancellations = [CancellationResult(oid, False) for oid in order_id_set]
+        return successful_cancellations + failed_cancellations
+
+    async def cancel_all_account_orders(self, trading_pair: str):
+        try:
+            ex_symbol = await CoinflexPerpetualAPIOrderBookDataSource.convert_to_exchange_trading_pair(
+                hb_trading_pair=trading_pair,
+                domain=self._domain,
+                throttler=self._throttler,
+                api_factory=self._api_factory
+            )
+            response = await self._api_request(
+                path=CONSTANTS.CANCEL_ALL_OPEN_ORDERS_URL.format(ex_symbol),
+                method=RESTMethod.DELETE,
+                is_auth_required=True,
+                limit_id=CONSTANTS.CANCEL_ALL_OPEN_ORDERS_URL,
+            )
+            if response.get("code") == 200:
+                for order_id in list(self._client_order_tracker.active_orders.keys()):
+                    self.stop_tracking_order(order_id)
+            else:
+                raise IOError(f"Error canceling all account orders. Server Response: {response}")
+        except Exception as e:
+            self.logger().error("Could not cancel all account orders.")
+            raise e
```

Comment:
There are no senders of `cancel_all_account_orders` in any strategy base code. This method is not required.